config/config.go

```diff
@@ -95,10 +95,13 @@ type WarpQuorum struct {
 }
 
 type Config struct {
-	LogLevel               string                   `mapstructure:"log-level" json:"log-level"`
-	PChainAPIURL           string                   `mapstructure:"p-chain-api-url" json:"p-chain-api-url"`
-	InfoAPIURL             string                   `mapstructure:"info-api-url" json:"info-api-url"`
-	StorageLocation        string                   `mapstructure:"storage-location" json:"storage-location"`
+	LogLevel        string `mapstructure:"log-level" json:"log-level"`
+	PChainAPIURL    string `mapstructure:"p-chain-api-url" json:"p-chain-api-url"`
+	InfoAPIURL      string `mapstructure:"info-api-url" json:"info-api-url"`
+	StorageLocation string `mapstructure:"storage-location" json:"storage-location"`
+	APIPort         uint64 `mapstructure:"api-port" json:"api-port"`
+	MetricsPort     uint64 `mapstructure:"metrics-port" json:"metrics-port"`
```

Comment:
These were added to support running multiple relayers on the same host in the new E2E test. They are optional, and will default to the same values that are currently used.